### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by coloratacirca** [JonasBernard/FakeStandby#69]
**Describe the bug**
When fake standby is started the screen goes black but only for 30 seconds, then it goes back to normal. Secure mode is not on and I gave it all the permission the app asked + permission for working in background and permission to ignore battery optimization.

**To Reproduce**
I just installed the apk from F-Droid, followed what the app said about the permission and such and tried to execute it. I also tried with downloading the apk from github (you never know) but same result.

**Expected behavior**
As far as I understood, the screen should stay black indefinitely

**Device information:**
 - Device: Realme 7 pro
 - Android Version: 12
 - App Version 1.4.0

**Comment by coloratacirca** [comment]
I also tried using an older version (1.3.3 as it was suggested in another issue) and the same problem happens, so I guess it's an issue with my phone and/or settings

**Comment by justincys** [comment]
Did you make sure that Secure mode is disabled in the FakeStandby settings?

**Comment by coloratacirca** [comment]
yes I did. With secure mode on the screen goes back to normal after 15 seconds, as it shod be. With Secure mode turned off, the screen goes back to normal after 30 seconds

**Comment by E0697** [comment]
Same here. Same device and OS.

**Comment by E0697** [comment]
This seems related to the screen-off time and the period changes accordingly.

**Comment by JonasBernard** [comment]
I did some research on that. It seems like the timeout habdler I use has unexpected behaviour, when initialising using the constructor without any arguments. So I will (as soon as I find the time) add the recommend argument and publish a new version.

**Comment by rak-rak** [comment]
Same issue on Samsung S22+, Android 13

Display wakes up after 30 seconds.

**Comment by rak-rak** [comment]
Hi, more clarification to that issue:
"30 seconds" is the time that is set in Android until the display turns off. 
If you set this to e.g. "2 minutes" and use FakeStandby, the screen will be enabled after2 minutes. Weird.

That means:
Using FakeStandby the display will turn on after the time that is set in Android to turn the display off. (Samsung Android 13).

Workaround could be:
1. FakeStandby should set the time in Android for display turn off to a very big value.
2. Then show the black screen
3. After the user has turned the display on, FakeStandby should set the Android setting to previous value.
